### DISCUSSION TITLE
提示封面标题过长，什么设置能修改每行字数

### DISCUSSION THREAD

**Discussion by sastars9** [nju-lug/NJUThesis#188] [Q&A] [answered]
我的标题被自动分成三行，第三行只有两个字，能不能修改一下每行字数，把这两个字分到前两行去，一行比标准只多一个字应该不会影响吧？谢谢
![image](https://user-images.githubusercontent.com/128053942/226409093-7348ecd2-0ace-462d-81d8-0e50aefbe3c9.png)

**Comment by note286** [comment]
我记得可以手动换行，例如，`测试\\测试测试\\测试`。

**Reply by sastars9** [reply]
是的，但是这样做还是三行，因为每行字数有限，我就想多加一个字

**Reply by note286** [reply]
昨晚临睡觉前回复的，看的不仔细，不好意思，等作者回复你吧。

**Comment by atxy-blip** [comment, marked as answer]
导言区加下面的命令，可以把封面横线改成任意你喜欢的长度。

```tex
\njusetlength{crulewd}{300pt}
```

**Reply by sastars9** [reply]
实测好用，感谢
